api/src/main/java/com/basistech/util/TextDomain.java

```diff
@@ -149,29 +144,43 @@ void setTransliterationScheme(TransliterationScheme scheme) {
      * @return String representation of the TextDomain.
      */
     public String toString() {
-        return "[" + theScript.code4() + "/" + theLanguage.ISO639_3() + "/" + theScheme.getName() + "]";
+        return "["
+                + theScript.code4()
+                + "/"
+                + theLanguage.ISO639_3()
+                + "/"
+                + theScheme.getName()
+                + "]";
     }
 
-    /**
-     * {@inheritDoc}
-     */
     public int compareTo(TextDomain o) {
-        if (theScript.numeric() > o.getScript().numeric()) {
```

Comment:
You changed `compareTo` (just for formatting?) but there's not test that calls it.
Also, a release note describing the bug fix?